src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,49 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+
+const setId = id => `#${id}`;
+
+export const Tabs = ({
+  tabs,
+  onTabSelected,
+  selectedTabId,
+  table,
+}) => {
+  const checkChange = (tab) => {
```

Comment:
```suggestion
  const makeCheckChange = (tab) => () => {
```
consider to make function factory. it will avoid to make arrow function in JSX